converter/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -0,0 +1,166 @@
+	ACI_MEASURE_PERFORMED_RNR_MEASURE_PERFORMED_REPEATED("ACI Measure Performed RnR's Measure Performed can only be present once"),
+	ACI_MEASURE_PERFORMED_RNR_MEASURE_ID_NOT_SINGULAR("ACI Measure Performed RnR's requires a single Measure ID"),
+	DENOMINATOR_COUNT_INVALID("Denominator count must be less than or equal to Initial Population count for an eCQM that is proportion measure"),
+	POPULATION_CRITERIA_COUNT_INCORRECT("The eCQM (electronic measure id: %s) requires %d %s(s) but there are %d", true),
+	ACI_NUMERATOR_DENOMINATOR_PARENT_NOT_ACI_SECTION("ACI Numerator Denominator Node should have an ACI Section Node as a parent"),
+	ACI_NUMERATOR_DENOMINATOR_MISSING_MEASURE_ID("ACI Numerator Denominator Node does not contain a measure name ID"),
+	ACI_NUMERATOR_DENOMINATOR_MISSING_CHILDREN("ACI Numerator Denominator Node does not have any child Nodes"),
+	ACI_NUMERATOR_DENOMINATOR_VALIDATOR_MISSING_DENOMINATOR_CHILD_NODE("This ACI Numerator Denominator Node does not contain a Denominator Node child"),
+	ACI_NUMERATOR_DENOMINATOR_VALIDATOR_MISSING_NUMERATOR_CHILD_NODE("This ACI Numerator Denominator Node does not contain a Numerator Node child"),
+	ACI_NUMERATOR_DENOMINATOR_VALIDATOR_TOO_MANY_DENOMINATORS("This ACI Numerator Denominator Node contains too many Denominator Node children"),
+	ACI_NUMERATOR_DENOMINATOR_VALIDATOR_TOO_MANY_NUMERATORS("This ACI Numerator Denominator Node contains too many Numerator Node children"),
+	ACI_SECTION_MISSING_REPORTING_PARAMETER_ACT("The ACI Section must have one Reporting Parameter ACT"),
+	CLINICAL_DOCUMENT_MISSING_ACI_OR_IA_OR_ECQM_CHILD("Clinical Document Node must have at least one Aci or IA or eCQM Section Node as a child"),
+	CLINICAL_DOCUMENT_MISSING_PROGRAM_NAME("Clinical Document must have one and only one program name"),
+	CLINICAL_DOCUMENT_INCORRECT_PROGRAM_NAME("Clinical Document program name is not recognized"),
+	CLINICAL_DOCUMENT_CONTAINS_DUPLICATE_ACI_SECTIONS("Clinical Document contains duplicate ACI sections"),
+	CLINICAL_DOCUMENT_CONTAINS_DUPLICATE_IA_SECTIONS("Clinical Document contains duplicate IA sections"),
+	CLINICAL_DOCUMENT_CONTAINS_DUPLICATE_ECQM_SECTIONS("Clinical Document contains duplicate eCQN sections"),
+	REPORTING_PARAMETERS_MUST_CONTAIN_SINGLE_PERFORMANCE_START("Must have one and only one performance start"),
+	REPORTING_PARAMETERS_MUST_CONTAIN_SINGLE_PERFORMANCE_END("Must have one and only one performance end"),
+	REPORTING_PARAMETERS_MISSING_PERFORMANCE_YEAR("Must have a performance year"),
+	QUALITY_MEASURE_SECTION_REQUIRED_REPORTING_PARAM_REQUIREMENT("The Quality Measure Section must have only one Reporting Parameter ACT"),
+	PERFORMANCE_RATE_INVALID_VALUE("Must enter a valid Performance Rate value"),
+	CPC_CLINICAL_DOCUMENT_MISSING_PRACTICE_SITE_ADDRESS("Must contain a practice site address for CPC+ conversions"),
+	CPC_CLINICAL_DOCUMENT_ONLY_ONE_APM_ALLOWED("One and only one Alternative Payment Model (APM) Entity Identifier should be specified"),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 134 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)